### ISSUE TITLE
HTTPS ability for snapshots, webRTC

### ISSUE THREAD

**Issue by ofadam** [mrlt8/docker-wyze-bridge#702]
Would it be possible to use the same keys/certs already in use for RTSP_HLSSERVERKEY/CERT to allow HTTPS webRTC and snapshots?

**Comment by mrlt8** [comment]
Potentially, but it may be better to put the bridge behind a more robust reverse proxy?

**Comment by ofadam** [comment]
True... although I love the idea of using only this repo to handle everything (avoiding the separate parts it would otherwise require.)
